iroha/src/tx.rs

```diff
@@ -11,14 +11,11 @@ use std::{
     time::{Duration, SystemTime},
 };
 
-/// This structure represents transaction in non-trusted form.
-///
-/// `Iroha` and its' clients use `RequestedTransaction` to send transactions via network.
-/// Direct usage in business logic is strongly prohibited. Before any interactions
-/// `accept`.
+/// This structure represents transaction right after construction.
 #[derive(Clone, Debug, Io, Encode, Decode)]
-pub struct RequestedTransaction {
+pub struct CreatedTransaction {
```

Comment:
`CreatedTransaction` doesn't sounds good in my opinion. Maybe as it's the first stage in the Transaction lifecycle and the only one with `new` we may name it just `Transaction`?